modules/openapi-generator/src/main/java/org/openapitools/codegen/serializer/OpenAPISerializer.java

```diff
@@ -14,35 +14,36 @@ public class OpenAPISerializer extends JsonSerializer<OpenAPI> {
     public void serialize(OpenAPI value, JsonGenerator gen, SerializerProvider serializers) throws IOException {
         if (value != null) {
             gen.writeStartObject();
-            gen.writeStringField("openapi", value.getOpenapi());
-            if(value.getInfo() != null) {
-                gen.writeObjectField("info", value.getInfo());
-            }
-            if(value.getExternalDocs() != null) {
-                gen.writeObjectField("externalDocs", value.getExternalDocs());
-            }
-            if(value.getServers() != null) {
-                gen.writeObjectField("servers", value.getServers());
-            }
-            if(value.getSecurity() != null) {
-                gen.writeObjectField("security", value.getSecurity());
-            }
-            if(value.getTags() != null) {
-                gen.writeObjectField("tags", value.getTags());
-            }
-            if(value.getPaths() != null) {
-                gen.writeObjectField("paths", value.getPaths());
-            }
-            if(value.getComponents() != null) {
-                gen.writeObjectField("components", value.getComponents());
-            }
-            if(value.getExtensions() != null) {
-                for (Entry<String, Object> e : value.getExtensions().entrySet()) {
-                    gen.writeObjectField(e.getKey(), e.getValue());
-                }
-            }
+            writeFieldIfNotNull(gen, "openapi", value.getOpenapi());
```

Comment:
Before, this was executed even if `value.getOpenapi()==null`. Now it won't be executed anymore in that case. Is this change on purpose?

Reply:
Yes I have changed it on purpose, to execute and write on field only when not null. If that does fit the central idea i can change that